internal/dataplane/parser/translate_tlsroute.go

```diff
@@ -29,11 +29,6 @@ func (p *Parser) ingressRulesFromTLSRoutes() ingressRules {
 
 	var errs []error
 	for _, tlsroute := range tlsRouteList {
-		if p.featureFlags.ExpressionRoutes {
-			p.registerResourceFailureNotSupportedForExpressionRoutes(tlsroute)
-			continue
-		}
-
 		if err := p.ingressRulesFromTLSRoute(&result, tlsroute); err != nil {
```

Comment:
Need to check kong version from parser here. If the KongVersion is before `ExpressionRouterL4Cutoff` (3.4.0), Kong does not support L4 proxy with expression router, so we still should register a translation failure here.

Reply:
Ok, I will fix it.